### DISCUSSION TITLE
slow performance on  request

### DISCUSSION THREAD

**Discussion by treeindark** [symfony/symfony#50531] [Q&A] [answered]
![image](https://github.com/symfony/symfony/assets/405997/e7e07d90-41ad-400b-af72-2d08743a3ac0)

I've this on API request but I don't know what it refer ????

controller.get_arguments 
ServiceValueResolver::resolve

Someone know this ?

**Comment by a-vallet** [comment]
ServiceValueResolver::resolve

I'm not very experienced but it seems to me that it is the autowiring Services into controller params. Must have a problem with a param ?

**Comment by stof** [comment]
Can you share the signature of your constructor for which this happened, to try to understand what's wrong with it ?

**Comment by treeindark** [comment, marked as answer]
Route Constructor have a Service Stuart, (from stuartapp/stuart-client-php), ok without, I will trace this!, thank you
